vertical-pod-autoscaler/docs/features.md

```diff
@@ -16,3 +17,19 @@ with VPA resource policy, VPA will follow VPA policy (and set values outside the
 range).
 
 To disable getting VPA recommendations for an individual container, set `mode` to `"Off"` in `containerPolicies`.
+
+## Memory Value Humanization
+
+VPA can present memory recommendations in human-readable binary units (KiB, MiB, GiB, TiB) instead of raw bytes, making resource recommendations easier to understand. This feature is controlled by the `--humanize-memory` flag in the recommender component.
+
+When enabled, memory values in recommendations will be:
+- Converted to the most appropriate binary unit (KiB, MiB, GiB, or TiB)
+- Displayed with up to 2 decimal places for precision
```

Comment:
Is it worth mentioning that this feature effectively rounds the recommendation up a little, meaning that the recommendation could be artificially bloated a little?

Reply:
Added